src/extensions/extension-loader.ts

```diff
@@ -271,7 +271,7 @@ export class ExtensionLoader extends Singleton {
 
   loadOnClusterManagerRenderer() {
     logger.debug(`${logModule}: load on main renderer (cluster manager)`);
-    this.autoInitExtensions(async (extension: LensRendererExtension) => {
+    const bundledLoaded = this.autoInitExtensions(async (extension: LensRendererExtension) => {
```

Comment:
nit: why not just `return this.autoInitExtensions(...`

Reply:
yeah might as well. done